pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -437,13 +439,20 @@ func (s *Service) runInstance(role string, i *v1alpha1.Instance) (*v1alpha1.Inst
 		return nil, errors.Errorf("no instance returned for reservation %v", out.GoString())
 	}
 
-	s.scope.V(2).Info("Waiting for instance to run", "instance-id", *out.Instances[0].InstanceId)
-	err = s.scope.EC2.WaitUntilInstanceRunningWithContext(
-		aws.BackgroundContext(),
+	waitTimeout := 1 * time.Minute
```

Comment:
Should this timeout be longer than 1 minute?

Reply:
I have seen that when this function actually works, 1m is more than enough to get the signal back. I'm also open to completely remove this function given that we can rely on reconciliation to take care of it

Reply:
We need to be careful not to make assumptions related to the Regions/Zones we are frequently testing in. From my experience not all AWS Regions are as responsive as us-east-* and us-west-*